### ISSUE TITLE
Positive carpal Tinel sign

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6725]
**Preferred term label:**
Positive carpal Tinel sign

**Synonyms**
Positive carpal Hoffmann Tinel sign

**Definition (free text, please give PubMed ID)**
The Tinel test is performed by lightly tapping (percussing) over the median nerve. It is positive (abnormal) if the patient experiences pain and paresthesias (tingling, numbness) along the distribution of the median nerve.

PMID:32310394
PMID:21524035

Carpal tunnel syndrome is the most common entrapment neuropathy. The pain and paresthesias characterizing carpal tunnel syndrome are typically along the distribution of the median nerve (i.e., thumb, index finger, and middle finger). At times, these symptoms may occur in all the fingers, but should not occur in the dorsum or palm of the hand. 

**Parent term (use hpo.jax.org/app)**
Abnormal peripheral nervous system physiology HP:0032120

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
OMIM:619161 

13/14  PMID:32686688

**Comment by pnrobinson** [comment]
added
